osmosis-1/osmosis.zone_assets.json

```diff
@@ -2719,6 +2719,24 @@
       "path": "transfer/channel-20100/nscr",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "source",
+      "base_denom": "ibc/C97473CD237EBA2F94FDFA6ABA5EC0E22FA140655D73D2A2754F03A347BBA40B",
+      "path": "transfer/channel-8945/ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A",
+      "osmosis_verified": false,
+      "osmosis_unlisted": true,
+      "override_properties": {
```

Comment:
These overrides are not required.

Reply:
Is the "canonical" section required or should that be removed as well?

Reply:
The canonical section can stay, because it is non-standard that we'd use a multihop IBC token, but I can see why it makes sense to do here